src/server/Document/component.jsx

```diff
@@ -43,7 +43,19 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
           </Fragment>
         )}
         {data.isAmp && (
-          <script key="amp" async src="https://cdn.ampproject.org/v0.js" />
+          <Fragment>
+            <script key="amp" async src="https://cdn.ampproject.org/v0.js" />
+            <script
+              async
+              custom-element="amp-geo"
+              src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"
+            />
+            <script
+              async
+              custom-element="amp-consent"
+              src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"
+            />
```

Comment:
Is there any way we can conditionally load these only for first time users? (Return users who have their cookies set already would not need to see the banner and wouldn't need the JS)

Micro-optimisation, I know!

Reply:
Nope, amp doesnt have any client side JS, everything you could ever need needs to be loaded

Reply:
> amp doesnt have any client side JS

Presumably you mean __`amp-geo`__ doesn't have any client-side JS. AMP itself _is_ client-side JS! 😄

Reply:
Ah yes soz, I meant amp doesnt have any simorgh client side js :P